docs/spec.md

```diff
@@ -949,23 +949,25 @@ def batch_norm_grad(operand, scale, mean, variance, grad_output, epsilon, featur
   # Use the implementation from batchnorm_expander.cc in XLA
   # Temporary variables have exactly the same names as in the C++ code
   elements_per_feature = constant(
-    divide(size(operand), dim(operand, feature_index)))
+    divide(num_elements(operand), dim(operand, feature_index)))
   i1 = multiply(
     grad_output,
     broadcast_in_dim(elements_per_feature, [], shape(operand)))
   i2 = broadcast_in_dim(
-    compute_sum(grad_output, feature_index),
+    compute_sum(grad_output),
```

Comment:
Not sure I understand the reasoning behind this change.

Reply:
Not sure what happened either. I meant to add `feature_index` for `i3`.